docs/general/administration/migrate.md

```diff
@@ -5,9 +5,15 @@ title: Migrating
 
 # Migrating
 
-It is possible to migrate your system to another system by using environment variables.
-It's possible to do this via the command line or by using Docker environment variables.
-To read more, see the [Configuration](/docs/general/administration/configuration) page.
+This page explains how to migrate from Emby to Jellyfin as well as how to migrate a Jellyfin installation.
+
+Jellyfins internal databases are not designed for migrations. Depending on your case there may be ways to work around this, f.ex. by just migrating parts of the data, or because it's possible maintain the same file paths.
+
+If that's not an option for you, there is also a script available to migrate the entire database without data loss, but it's not a turn-key solution.
+
+## Full Database Migration
+
+[Jellyfin Migrator](https://github.com/MMMZZZZ/Jellyfin-Migrator) is a script that can migrate your entire installation from Windows to Linux or Linux Docker. It can also do other migrations, like f.ex. a reorganization of your media files. All statistics, settings and metadata are preserved. Please check the readme of the project for further details.
```

Comment:
```suggestion
[Jellyfin Migrator](https://github.com/MMMZZZZ/Jellyfin-Migrator) is a third-party script that can migrate your entire installation from Windows to Linux or Linux Docker. It can also do other migrations, like a reorganization of your media files. All statistics, settings and metadata are preserved.
```

- Clearly state that it's third-party to avoid users opening issues in our repositories
- Don't abbreviate "for example" or just remove it
- No need to instruct the user to read the readme. They'll do that anyway if they need to figure out how the tool works :p

Reply:
> Clearly state that it's third-party to avoid users opening issues in our repositories

Originally, I had that included. But then I saw that the page links to a couple other (apparently) third-party scripts as well without any such note. Therefore I decided to include such a note in the projects readme instead. 
I also opted against adding a disclaimer for all projects linked here because I felt it would disturb the reading more than it helps - especially since this doesn't seem to have been an issue for the other projects?

Other two points are fine for me; going to adjust them.

Reply:
> but I'd still suggest to add it to all third-party scripts then

I agree, this should happen in other places as well.